docs/build-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+TEMP_VENV_ENV="/tmp/sphinx"
```

Comment:
Renaming this variable to something more unique may be a good idea. Perhaps even switching to lower case.
Furthermore I think the directory name shouldn't be plain `sphinx`. It's very generic and may contain unrelated files.

Perhaps it could be generated with `mktemp` instead? We would loose some efficiency in reruns, but `/tmp` contents are ephemeral anyway. Plus we wouldn't have to deal with leftovers from previous runs.

Reply:
Totally agree on this one. `mktemp` is the ideal solution IMHO